addOns/reports/src/main/java/org/zaproxy/addon/reports/automation/OutputSummaryJob.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.reports.automation;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import net.sf.json.JSONObject;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.control.Control;
+import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.addon.automation.AutomationEnvironment;
+import org.zaproxy.addon.automation.AutomationJob;
+import org.zaproxy.addon.automation.AutomationProgress;
+import org.zaproxy.addon.automation.jobs.PassiveScanJobResultData;
+import org.zaproxy.addon.automation.jobs.PassiveScanJobResultData.RuleData;
+import org.zaproxy.addon.reports.ExtensionReports;
+
+public class OutputSummaryJob extends AutomationJob {
+
+    public static final String JOB_NAME = "outputSummary";
+
+    private static final String PARAM_FORMAT = "format";
+    private static final String PARAM_SUMMARY_FILE = "summaryFile";
+
+    private ExtensionReportAutomation ext;
+    private PrintStream out = System.out;
+
+    private enum Format {
+        NONE,
+        SHORT,
+        LONG
+    };
+
+    private Format format = Format.NONE;
+    private String summaryFile;
+
+    private ExtensionReports extReport;
+
+    public OutputSummaryJob(ExtensionReportAutomation ext) {
+        this.ext = ext;
+    }
+
+    /**
+     * Prints a summary to std out as per the packaged scans. The output is deliberately not
+     * internationalised as neither was the output of the packaged scans.
+     */
+    @Override
+    public void runJob(
+            AutomationEnvironment env, LinkedHashMap<?, ?> jobData, AutomationProgress progress) {
+        if (Format.NONE.equals(format)) {
+            return;
+        }
+
+        int pass = 0;
+        int warn = 0;
+
+        // Number of URLs, as per logic behind the core.urls API endpoint
+        int numUrls = ext.countNumberOfUrls();
+        if (numUrls == 0) {
+            out.println(
+                    "No URLs found - is the target URL accessible? Local services may not be accessible from a Docker container");
+
```

Comment:
Could be removed.

Reply:
No, thats what the baseline says so this is also for backwards compatibility :)
https://github.com/zaproxy/zaproxy/blob/main/docker/zap-baseline.py#L356

Reply:
I just meant the blank line 😉

Reply:
Ha! I didnt even notice it ;)